app/geo/gui/src/ts/twxplore/gui/geo/states/map/initialMapState.ts

```diff
@@ -29,6 +21,7 @@ Object.values(FeatureType).map(featureType => {
     dirty: false,
     featureTypeState: MapFeatureTypeState.ABSENT_ON_MAP,
     attributeStates: {},
+    visible: null,
```

Comment:
Why do you need null? Why not default to false?

Reply:
Null was to indicate absence from the map completely (i.e. a feature of that type has yet to be added/loaded).
Changed by defaulting to false and using MapFeatureTypeState.ABSENT_ON_MAP instead.